scripts/configure-hypervisor.sh

```diff
@@ -192,6 +194,17 @@ show_array() {
 
 generate_qemu_options() {
 	#---------------------------------------------------------------------
+	#check if cross-compile is needed
+	host=$(uname -m)
+	if [ $arch != $host ];then
+		case $arch in
+			aarch64) qemu_options+=(size:--cross-prefix=aarch64-linux-gnu-);;
+			ppc64le);;
```

Comment:
die "Not not implemented"  ?

Reply:
@jcvenegas I'm not sure the prefix for ppc64le and s390, do you have any idea?

Reply:
@nitkon and @alicefr - could you help here?

Reply:
@jongwu  no idea :/ but just to make explicit that is not implemented you can add a way to exit so in the future could be added